script.js

```diff
@@ -64,3 +64,41 @@ function getCounty(lat, lon) {
 // Test cases
 // getCounty(0, 0);
 // getCounty(47.487983, -121.723172);
+
+
+function popularity (trailArray) {
```

Comment:
Also function naming convention, (not only in JS, but general programming convention), we name the function using a verb and name should be clear about what this function does. e.g. `function updateTrialPopularity(..){...}`